### PR TITLE
Adding cucumber 2.3.1 angles adapter

### PR DESCRIPTION
**Pull request by cosminoance**
Adding cucumber 2.3.1 Angles adapter.
### Cucumber 2
- Extend the step definition file with: 
    ```java
    public class StepDef extends AnglesCucumber2Adapter
    ```
    - Include `AnglesCucumber2Adapter` as a plugin in the *run with* class if you run through `Maven`
        ```java
        @CucumberOptions(plugin = { 
              "com.github.angleshq.angles.listeners.cucumber.AnglesCucumber2Adapter"})   
        ```
- Inside the `step definition` class, you need to include an empty `@Before` method to allow instantiation of the `Angles` reporter.
    ```java
  @Before
    public static void initilizeAnglesAdapter() {
    }
    ```

### CODE REVIEW

File: src/main/java/com/github/angleshq/angles/listeners/cucumber/AnglesCucumber2Adapter.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+                    PickleTable argument = (PickleTable) testStep.getStepArgument().get(0);
+                    for (PickleRow row : argument.getRows()) {
+                        argumentTable += "\n| ";
+                        for (PickleCell cell : row.getCells()) {
+                            argumentTable += cell.getValue() + " | ";
+                        }
+                    }
+                    anglesReporter.info(argumentTable);
+                } catch (Exception e) {
+
```
Comment (snevesbarros) [inline src/main/java/com/github/angleshq/angles/listeners/cucumber/AnglesCucumber2Adapter.java]: Maybe add an info here, that we weren't able to display the table?

File: src/main/java/com/github/angleshq/angles/listeners/cucumber/AnglesCucumber2Adapter.java [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+            //further split the path to get just the feature file name itself
+            featurePath = featureName.split("/");
+            featureName = featurePath[featurePath.length-1];
+            String testName = event.testCase.getName();
+            anglesReporter.startTest(featureName, testName);
+        }
+    };
+
+    protected EventHandler<TestCaseFinished> handleTestCaseFinished =
+            new EventHandler<TestCaseFinished>() {
```
Comment (snevesbarros) [inline src/main/java/com/github/angleshq/angles/listeners/cucumber/AnglesCucumber2Adapter.java]: indentation too much?

**Review by snevesbarros — CHANGES_REQUESTED** [review]
just some minor changes, but looking good.

File: src/main/java/com/github/angleshq/angles/listeners/cucumber/AnglesCucumber2Adapter.java (hunk 2 above)
Comment (cosminoance) [reply src/main/java/com/github/angleshq/angles/listeners/cucumber/AnglesCucumber2Adapter.java]: corrected

File: src/main/java/com/github/angleshq/angles/listeners/cucumber/AnglesCucumber2Adapter.java (hunk 1 above)
Comment (cosminoance) [reply src/main/java/com/github/angleshq/angles/listeners/cucumber/AnglesCucumber2Adapter.java]: added

**Comment by cosminoance** [comment]
changes requested committed

**Review by snevesbarros — APPROVED** [review]

**Pull request merged**